recipes/zeromq/all/conanfile.py

```diff
@@ -17,14 +18,20 @@ class ZeroMQConan(ConanFile):
         "fPIC": [True, False],
         "encryption": [None, "libsodium", "tweetnacl"],
         "with_norm": [True, False],
-        "poller": [None, "kqueue", "epoll", "devpoll", "pollset", "poll", "select"]
+        "poller": [None, "kqueue", "epoll", "devpoll", "pollset", "poll", "select"],
+        "with_draft_api": [True, False],
+        "with_websocket": [True, False],
```

Comment:
does it require websocket library?

Reply:
I think zeromq has original implementation for websocket. 
https://github.com/zeromq/libzmq/blob/5eb0b00c875b8406f430573bdaf9eab5d4a0c99e/CMakeLists.txt#L177-L216